modules/pel/peltool/peltool.py

```diff
@@ -472,6 +511,10 @@ def main():
                         action='store_true', help='Show number of PELs')
     parser.add_argument('-r', '--reverse',
                         action='store_true', help='Reverse order of output')
+    parser.add_argument('-c', '--clear',
+                        dest='IDToDelete', help='Delete a PEL based on its ID')
+    parser.add_argument('-C', '--delete-all', dest='deleteAll',
```

Comment:
Ramesh and me discussed and we are thinking maybe we can remove current --directory option and merge the functionality with -p option. As -p option is only available in non-BMC environment, this will resolve all our concerns.

Reply:
We only have 1 user of the -d option, but it's an important one that we can't break. Fortunately, I don't think they are doing refreshes from the Artifactory via an automatic mechanism.  So if we don't do the staging like I proposed above, I'll have to reach out to them after we have the new code in the artifactory and I can work with them to change their scripts that call this.

Reply:
With '-p' to specify the path, we'd still need an option to specify the 'write every parsed PEL in that directory to a file named after the PEL file'.